### PR TITLE
feat: ✨ quote pattern

### PR DESCRIPTION
**Pull request by yoezlem**
This component is currently a draft. We want to take a look at it together and assess whether it makes sense to create sd-quote as a sample or not.

Edit: Not anymore. So feel free to review it ;)

### CODE REVIEW

**Comment by mariohamann** [comment]
That would be a pattern. And therefore it shouldn't have any API with different styles and should just be called `quote` but not `sd-quote`. Just copyable code and nothing else.

**Comment by mariohamann** [comment]
Nice! That looks pretty fantastic.

Some small stuff:

CleanShot 2023-09-15 at 16 46 52@2x

Please use our default focus style for this element. I think it's something like `focus-visible:focus-outline` or similar.

CleanShot 2023-09-15 at 16 47 50@2x

Please remove the accordion thing here, as this shouldn't be used from a UX perspective (even if it's technically possible).

CleanShot 2023-09-15 at 16 48 41

Please make a fixed/bigger width for nav items, so it doesn't jump when collapsing.

CleanShot 2023-09-15 at 16 50 12@2x

We use semantic versioning, so we're just doing a minor bump. Just wait until you finally merge and make a single commit beofre that, which uses e. g. `1.14` or whatever is the next version than.

---

@coraliefeil Currently the icon-only option is missing – I would like to make this as an improvement within a next ticket, so Alec can can close this one here sooner.

---

@van-nguyen-ht If you have updated on the "current" line on the left, we would need that now. :)

**Comment by yoezlem** [comment]
@mariohamann wrong component I guess..? 😉

> Nice! That looks pretty fantastic.
> 
> Some small stuff:
> 
> CleanShot 2023-09-15 at 16 46 52@2x
> * [ ]   Please use our default focus style for this element. I think it's something like `focus-visible:focus-outline` or similar.
> 
> CleanShot 2023-09-15 at 16 47 50@2x
> * [ ]  Please remote the  size part in the `variant x size` story from the accordion, as this shouldn't be used from a UX perspective (even if it's technically possible).
> 
> CleanShot 2023-09-15 at 16 48 41 [ CleanShot 2023-09-15 at 16 48 41 ](https://user-images.githubusercontent.com/26542182/268315857-aecb112f-0b78-47cf-be66-5d2a9fcf041a.gif) [ ](https://user-images.githubusercontent.com/26542182/268315857-aecb112f-0b78-47cf-be66-5d2a9fcf041a.gif)
> 
> * [ ]  Please make a fixed/bigger width for nav items, so it doesn't jump when collapsing.
> 
> CleanShot 2023-09-15 at 16 50 12@2x
> * [ ]  We use semantic versioning, so we're just doing a minor bump. Just wait until you finally merge and make a single commit beofre that, which uses e. g. `1.14` or whatever is the next version than.
> 
> * [ ]  @coraliefeil Currently the icon-only option is missing – I would like to make this as an improvement within a next ticket, so Alec can can close this one here sooner.
> 
> * [ ]  @van-nguyen-ht If you have updated on the "current" line on the left, we would need that now. :)

**Comment by coraliefeil** [comment]
I approved the PR ... 

Though in the default the quote text color needs to be changed to #181818.
https://64469bd84c5ea1ce2dd5d016-cxcubrggor.chromatic.com/?path=/story/pattern-quote--default
In the stories Size and Variant the intended font color #181818 is used. 

I assume for sd-paragraph the font size fix is done separately?

Thanks <3

**Comment by yoezlem** [comment]
@coraliefeil Thanks, I changed to color for the default quote.

And yes, we will fix the font-size for sd-paragraph in a seperate Ticket: issue-406 (https://github.com/solid-design-system/solid/issues/406)

File: packages/components/src/patterns/quote/quote.stories.ts
Comment (karlbaumhauer) [inline packages/components/src/patterns/quote/quote.stories.ts:27]: The description should be changed to something describing more the use of patterns (IMO). From what I understand there is no "versions" or "states" in patterns right?

File: packages/components/src/patterns/quote/quote.stories.ts
Comment (karlbaumhauer) [inline packages/components/src/patterns/quote/quote.stories.ts:46]: As in l27.

File: packages/components/src/patterns/quote/quote.stories.ts
Comment (karlbaumhauer) [inline packages/components/src/patterns/quote/quote.stories.ts:94]: As in l27.

File: packages/components/src/patterns/quote/quote.stories.ts
Comment (karlbaumhauer) [inline packages/components/src/patterns/quote/quote.stories.ts:142]: As in l27.

**Review by karlbaumhauer — CHANGES_REQUESTED** [review]

File: packages/components/src/patterns/quote/quote.stories.ts
Comment (karlbaumhauer) [reply packages/components/src/patterns/quote/quote.stories.ts:27]: like the idea...

Comment (mariohamann) [reply packages/components/src/patterns/quote/quote.stories.ts:27]: Sounds great! Some days ago we discussed the difference between samples and patterns and it feels like this could be the right moment to write that finally down.

I could take that over today!

**Review by Vahid1919 — APPROVED** [review]

**Review by azraefendic — APPROVED** [review]

**Pull request merged**
